### PR TITLE
Visual edits

### PR DESCRIPTION
**Pull request by Crissman**
From our discussion of the visual Notebooks.

### CODE REVIEW

**Comment by Crissman** [comment]
Done the changes I feel we're straightforward. Please review!

File: examples/visualization/plot_parallel_coordinate.ipynb [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "from sklearn.datasets import fetch_openml\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "mnist = fetch_openml('mnist_784', version=1)\n",
+    "mnist = fetch_openml(name="Fashion-MNIST", version=1)\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_parallel_coordinate.ipynb]: ditto.
```suggestion
    "mnist = fetch_openml(name='Fashion-MNIST', version=1)\n",
```

File: examples/visualization/plot_optimization_history.ipynb [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "from sklearn.datasets import fetch_openml\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "mnist = fetch_openml('mnist_784', version=1)\n",
+    "mnist = fetch_openml(name="Fashion-MNIST", version=1)\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_optimization_history.ipynb]: ditto
```suggestion
    "mnist = fetch_openml(name='Fashion-MNIST', version=1)\n",
```

File: examples/visualization/plot_intermediate_values.ipynb [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "from sklearn.datasets import fetch_openml\n",
     "from sklearn.model_selection import train_test_split\n",
     "\n",
-    "mnist = fetch_openml('mnist_784', version=1)\n",
+    "mnist = fetch_openml(name="Fashion-MNIST", version=1)\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_intermediate_values.ipynb]: Please use single quotes to avoid parse errors.

```suggestion
    "mnist = fetch_openml(name='Fashion-MNIST', version=1)\n",
```

**Comment by Crissman** [comment]
Done. Thanks.

File: examples/visualization/plot_optimization_history.ipynb [hunk 4]
```diff
@@ -4,17 +4,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Optimization History in Jupyter Notebook\n",
+    "# Visualizing Optimization History\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
-    "After optimizing the hyperparameter of neural networks, `plot_optimization_history()` plots optimization history in a study."
+    "After optimizing the hyperparameter of neural networks, [``plot_optimization_history()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html) plots optimization history in a study."
```
Comment (toshihikoyanase) [inline examples/visualization/plot_optimization_history.ipynb]: '`' is duplicated.

```suggestion
    "After optimizing the hyperparameter of neural networks, [`plot_optimization_history()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html) plots optimization history in a study."
```

File: examples/visualization/plot_contour.ipynb [hunk 5]
```diff
@@ -64,7 +64,7 @@
     "\n",
     "    layers = []\n",
     "    for i in range(n_layers):\n",
-    "        n_units = int(trial.suggest_loguniform('n_units_l{}'.format(i), 32, 256))\n",
+    "        n_units = trial.suggest_int('n_units_l{}'.format(i), 32, 256)\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_contour.ipynb]: This is just a comment. `suggest_loguniform` was used to search the order of magnitude of `n_units` similarly to the [Chainer example](https://github.com/pfnet/optuna/blob/master/examples/chainer_simple.py#L45).

I would like to accept this change because of the following two reasons:
1. The search space is small enough (it has about two hundred choices), and
1. the objective functions in visualization examples are almost the same, and it can be merged into one. In that case, the parallel coordinate plot does not support the log-scale axis (see https://github.com/pfnet/optuna/issues/669).

Comment (Crissman) [reply examples/visualization/plot_contour.ipynb]: Understood. Thanks for the explanation,

File: examples/visualization/plot_contour.ipynb [hunk 6]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Hyper-parameter Relationship in Jupyter Notebook\n",
+    "# Visualizing Hyper-parameter Relationships\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
     "After optimizing the hyperparameter of neural networks, `plot_contour()` plots hyper-parameter relationship of completed trials in a study.\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_contour.ipynb]: IMO, it is helpful to provide a link to the reference like [`plot_intermediate_values.ipynb`](https://github.com/pfnet/optuna/pull/667/files#diff-c93a73ade9fdafd6e829fe52ee95b2a6R10).
```suggestion
     "After optimizing the hyperparameter of neural networks, [`plot_contour()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_contour) plots hyper-parameter relationship of completed trials in a study.\n",
```

File: examples/visualization/plot_intermediate_values.ipynb [hunk 7]
```diff
@@ -4,19 +4,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Learning Curves in Jupyter Notebook\n",
+    "# Visualizing Learning Curves\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
-    "After optimizing the hyperparameter of neural networks, `plot_intermediate_values()` plots learning curves of running, completed, and pruned trials in a study.\n",
+    "After optimizing the hyperparameter of neural networks, [`plot_intermediate_values()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html) plots learning curves of running, completed, and pruned trials in a study.\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_intermediate_values.ipynb]: IMO, adding the in-page link would be helpful for users.
```suggestion
    "After optimizing the hyperparameter of neural networks, [`plot_intermediate_values()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_intermediate_values) plots learning curves of running, completed, and pruned trials in a study.\n",
```

File: examples/visualization/plot_optimization_history.ipynb [hunk 8]
```diff
@@ -4,17 +4,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Optimization History in Jupyter Notebook\n",
+    "# Visualizing Optimization History\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
-    "After optimizing the hyperparameter of neural networks, `plot_optimization_history()` plots optimization history in a study."
+    "After optimizing the hyperparameter of neural networks, [`plot_optimization_history()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html) plots optimization history in a study."
```
Comment (toshihikoyanase) [inline examples/visualization/plot_optimization_history.ipynb]: ditto.
```suggestion
    "After optimizing the hyperparameter of neural networks, [`plot_optimization_history()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_optimization_history) plots optimization history in a study."
```

File: examples/visualization/plot_parallel_coordinate.ipynb [hunk 9]
```diff
@@ -4,10 +4,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing High-dimentional Parameter Relationships in Jupyter Notebook\n",
+    "# Visualizing High-dimensional Parameter Relationships\n",
     "\n",
     "This notebook demonstrates a visualization utility of Optuna.\n",
-    "After optimizing the hyperparameter of neural networks, `plot_parallel_coordinate()` plots high-dimentional parameter relationships in a study.\n",
+    "After optimizing the hyperparameter of neural networks, `plot_parallel_coordinate()` plots high-dimensional parameter relationships in a study.\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_parallel_coordinate.ipynb]: IMO, it is helpful to provide a link to the reference like `plot_intermediate_values.ipynb`.
```suggestion
    "After optimizing the hyperparameter of neural networks, [`plot_parallel_coordinate()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_parallel_coordinate) plots high-dimensional parameter relationships in a study.\n",
```

File: examples/visualization/plot_slice.ipynb [hunk 10]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Visualizing Hyper-parameter Relationship in Jupyter Notebook\n",
+    "# Visualizing Hyper-parameter Relationship\n",
     "This notebook demonstrates a visualization utility of Optuna. After optimizing the hyperparameter of neural networks, `plot_slice()` plots hyper-parameter relationship of completed trials in a study.\n",
```
Comment (toshihikoyanase) [inline examples/visualization/plot_slice.ipynb]: ditto.
```suggestion
     "This notebook demonstrates a visualization utility of Optuna. After optimizing the hyperparameter of neural networks, [`plot_slice()`](https://optuna.readthedocs.io/en/stable/reference/visualization.html#optuna.visualization.plot_slice) plots hyper-parameter relationship of completed trials in a study.\n",
```

File: examples/visualization/plot_contour.ipynb [hunk 11]
```diff
@@ -74,8 +74,7 @@
     "\n",
     "def objective(trial):\n",
     "    model = L.Classifier(create_model(trial))\n",
-    "    optimizer = chainer.optimizers.Adam()\n",
-    "    optimizer.setup(model)\n",
+    "    optimizer = chainer.optimizers.Adam().setup(model)\n",
```
Comment (hvy) [inline examples/visualization/plot_contour.ipynb]: This is a detail so please feel free to ignore it but I believe it's quite rare to chain `setup` this way. It's perfectly valid.

**Comment by hvy** [comment]
Verified that all notebook run just fime.

File: examples/visualization/plot_contour.ipynb (hunk 11 above)
Comment (Crissman) [reply examples/visualization/plot_contour.ipynb]: Yes, I've been pushing for this shorter form since I worked to get the PRs done to enable it over a year ago. Denser is better. ;)

**Comment by Crissman** [comment]
@toshihikoyanase Github appears to be broken on the ability to add that suggestion. When I click the `Commit Suggestion` button, it says, `This diff has recently been updated. Refresh and try again.` This continues no matter how many times I refresh the page. I went ahead and made that change locally, and it shows it in the code changes, but doesn't show that I implemented your suggestion.

Please merge, and if it isn't reflected, I'll make another PR. 🙇‍♂️

**Comment by hvy** [comment]
Is the codecov failure also related to the organization change maybe?

**Comment by hvy** [comment]
Restarted the CI to verify the codecov failure.

**Comment by hvy** [comment]
@Crissman sorry, could you fix the conflict? The codecov by the way did not reproduce so I assume it was the organization change after all.
